app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
   attribute :post_counter, default: 0
```

Comment:
- [ ] It is required that a new user signed up should be asked to confirm his or her email. This feature has been implemented in this milestone. Kindly add that.

### Project requirements

![project-rqt](https://github.com/anderCM/blog-app/assets/26597432/c889a674-2a3f-4054-a041-c6cab10ab73a)